gateway-js/README.md

```diff
@@ -13,10 +13,12 @@ const { ApolloServer } = require("apollo-server");
 const { ApolloGateway } = require("@apollo/gateway");
```

Comment:
```suggestion
const { ApolloGateway, IntrospectAndCompose } = require("@apollo/gateway");
```

Reply:
@trevor-scheer  Thank you very much for the review, I completely forgot about the import.
I'll try to help with some more complex issues, thanks again.